test/e2e/backupScript_test.go

```diff
@@ -0,0 +1,31 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/cluster-etcd-operator/test/e2e/framework"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os/exec"
+	"testing"
+)
+
+func TestBackupScript(t *testing.T) {
+	clientSet := framework.NewClientSet("")
+	masterNodes, err := clientSet.CoreV1Interface.Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: "node-role.kubernetes.io/master"})
+	require.NoErrorf(t, err, "error while listing master nodes")
+
+	// bash one of the master nodes
+	debugNodeName := masterNodes.Items[0].Name
+	debugNodeAsArg := fmt.Sprintf("debug node/%s", debugNodeName)
+	err = exec.Command("oc", debugNodeAsArg).Run()
+	require.NoError(t, err)
+
+	// run debug host
+	err = exec.Command("chroot /host").Run()
+	require.NoError(t, err)
+
+	// run the /usr/local/bin/cluster-backup.sh on the node
+	err = exec.Command("/usr/local/bin/cluster-backup.sh").Run()
```

Comment:
I think this will run the script locally on the machine executing the test. Check how it's done in the other test:
https://github.com/openshift/cluster-etcd-operator/blob/master/test/e2e/etcdctl_test.go#L91-L94